posthog/api/property_definition.py

```diff
@@ -130,6 +133,7 @@ def get_queryset(self):
                             WHERE team_id = %(team_id)s AND name NOT IN %(excluded_properties)s
                              {name_filter} {numerical_filter} {search_query} {event_property_filter}
                             ORDER BY is_event_property DESC, query_usage_30_day DESC NULLS LAST, name ASC
+                            LIMIT {limit} OFFSET {offset}
```

Comment:
flyby....

I think by setting limit and offset manually here we only ever get one page.

DRF `LimitOffsetPagination` uses the queryset to count the results here https://github.com/encode/django-rest-framework/blob/master/rest_framework/pagination.py#L387

And then uses that count to decide whether there are more pages. 

It slices the queryset before loading the results. 

So if we add limit and offset to the queryset the count is always equal to one page. 

The TTFB on cloud for this endpoint is _wild_ though even so